packages/cyberstorm/src/components/Layout/LoginLayout/BetaLoginLayout.tsx

```diff
@@ -19,30 +25,42 @@ export function BetaLoginLayout() {
             <div>Login to Thunderstore Beta</div>
           </div>
           <div className={styles.loginButtons}>
-            <div className={styles.loginButton}>
-              <Button
-                colorScheme="blue"
-                label="Discord"
-                leftIcon={<FontAwesomeIcon icon={faDiscord} fixedWidth />}
-                paddingSize="large"
-              />
-            </div>
-            <div className={styles.loginButton}>
-              <Button
-                colorScheme="gray"
-                label="GitHub"
-                leftIcon={<FontAwesomeIcon icon={faGithub} fixedWidth />}
-                paddingSize="large"
-              />
-            </div>
-            <div className={styles.loginButton}>
-              <Button
-                colorScheme="red"
-                label="Ovewolf"
-                leftIcon={<OverwolfLogo />}
-                paddingSize="large"
-              />
-            </div>
+            {props.discordAuthUrl ? (
+              <div className={styles.loginButton}>
+                <a href={props.discordAuthUrl}>
+                  <PlainButton
```

Comment:
You can't place buttons inside anchor elements. (Well I guess you can, but it's illegal. You wouldn't place a button inside a car.)

Reply:
The `PlainButton` is not a button, but just a div that looks like a button. But IIRC the current button/link component implementation will be refactored in the near future™️ Consult designer on discord about the timeline/importance.

Reply:
Alright. Having `import { PlainButton } from "./Button";` seems like a footgun then but that's outside the scope of this PR.

This comment can be marked as resolved.

Reply:
True. We should have a refactor task for the button and link components somewhere. If not that needs to be figured out with grav